LambdaEngine/Source/Rendering/Core/Vulkan/TextureViewVK.cpp

```diff
@@ -92,6 +97,15 @@ namespace LambdaEngine
 		}
 	}
 
+	void TextureViewVK::InternalRelease()
+	{
+		if (m_ImageView != VK_NULL_HANDLE)
+		{
+			SAFERELEASE(m_Desc.pTexture);
```

Comment:
Does SAFERELEASE need to be within the if-statement? If it's nullptr, nothing will happen.